nucleus/glassfish-jul-extension/src/test/java/org/glassfish/main/jul/handler/GlassFishLogHandlerTest.java

```diff
@@ -164,10 +168,11 @@ public void roll() throws Exception {
         );
         handler.publish(new GlassFishLogRecord(Level.SEVERE, "File two, line two", false));
         Thread.sleep(MILLIS_FOR_PUMP);
+        final List<String> fileLines3 = Files.readAllLines(logFile.toPath());
```

Comment:
If the file doesn't exist, this will cause IOException, so the assertion will not make sense.